datahub-web-react/src/app/entity/shared/components/styled/search/EmbeddedListSearch.tsx

```diff
@@ -123,6 +124,7 @@ export const EmbeddedListSearch = ({
     shouldRefetch,
     resetShouldRefetch,
 }: Props) => {
+    const { shouldRefetchEmbeddedListSearch, setShouldRefetchEmbeddedListSearch } = useEntityContext();
```

Comment:
This scares me. We use embeddedlistserach outside of entity contexts - e.g. for showing teh result of an ingestion run

Reply:
Curious why we are needing this now

Reply:
yeah that's true but I make sure that I do a check for `if (shouldRefetchEmbeddedListSearch)` before doing anything and always assume these could be undefined so other places outside of the entity context shouldn't be affected.

the reason it was needed now is that when you click "add to assets" on a data product profile page (or domain or glossary term) then you don't see assets added in their entities tab until after a refresh of the whole page. This will update that list automatically for you for all of them.

Reply:
Nice- that makes sense